src/static/js/client/sidebar-search.js

```diff
@@ -325,6 +342,16 @@ export function addPageListeners() {
       }, settings.stoppedTypingDelay);
   });
 
+  info.searchInput.addEventListener('keydown', domEvent => {
+    if (domEvent.key === 'ArrowDown') {
+      const elem = info.results?.firstChild;
+      if (elem && !elem.classList.contains('wiki-search-no-results')) {
```

Comment:
OK I see why this is named `elem` and not `firstResult`, since, you know, we're determining whether or not it's a result at all, here...

But can we maybe use an inclusive instead of an exclusive one? Like, results [always have](https://github.com/hsmusic/hsmusic-wiki/blob/15bc0f97e630a1b1d13d0bd80096d81b9a65d7de/src/static/js/client/sidebar-search.js#L694) the class `wiki-search-result`, so let's check for the presence of that.

Can we also fold in the not-nullish check?

```suggestion
      if (elem?.classList.contains('wiki-search-result')) {
```